server/client.go

```diff
@@ -169,3 +193,10 @@ func (c *Client) validateResponse(resp *http.Response) error {
 
 	return nil
 }
+
+func (c *Client) logRequest(methodName string, r *http.Request, hasBody bool) {
+	if c.logRequests {
+		requestDump, _ := httputil.DumpRequest(r, hasBody)
+		c.logger.LogInfo("Request in "+methodName, "request", string(requestDump))
+	}
+}
```

Comment:
It seems odd to have the setting called `logRequests` and the action having a similar name `logRequest`. Maybe have the setting called `shouldLogRequests`?

Reply:
Good idea. Done :+1: